bin/ch/WScriptJsrt.cpp

```diff
@@ -235,14 +234,25 @@ JsValueRef WScriptJsrt::LoadScriptHelper(JsValueRef callee, bool isConstructCall
             {
                 size_t unused;
                 IfJsrtErrorSetGo(ChakraRTInterface::JsStringToPointerUtf8Copy(arguments[3], &fileName, &unused));
+                freeFileName = true;
             }
         }
 
+        if (!freeFileName)
+        {
+            sprintf_s(fileNameBuffer, "script%i.js", (int)sourceContext);
```

Comment:
this might or might not be module as it could be normal script without filename. Don't quite want to get another name in the path here.